### PR TITLE
common: add chain and token information

### PR DESCRIPTION
**Pull request by kayleegeorge**
Add chain and foreign token constants for account upgrade. 

Fix: Prevent send any-stable on testnet (no swapper / Uniswap pools).

### CODE REVIEW

File: packages/daimo-common/src/chain.ts [hunk 1]
```diff
@@ -1,31 +1,178 @@
...
+  baseUSDC,
+  baseWETH,
+  optimismSepoliaUSDC,
+  optimismSepoliaWETH,
+  optimismUSDC,
+  optimismWETH,
+  polygonSepoliaUSDC,
+  polygonUSDC,
+  polygonWETH,
+} from "./foreignToken";
```
Comment (dcposch) [inline packages/daimo-common/src/chain.ts:30]: 🚀

File: packages/daimo-common/src/chain.ts [hunk 2]
```diff
@@ -1,31 +1,178 @@
...
 
 export type AccountChain = {
   chainId: number;
   name: string;
-  nativeWETH: ForeignToken;
+  CCTPDomain: number;
```
Comment (dcposch) [inline packages/daimo-common/src/chain.ts:35]: `cctpDomain`

File: packages/daimo-common/src/chain.ts [hunk 3]
```diff
@@ -1,31 +1,178 @@
...
   name: "baseSepolia",
+  CCTPDomain: 6,
+  bridgeCoin: baseSepoliaUSDC,
   nativeWETH: baseSepoliaWETH,
+  isTestnet: true,
+};
+
+export const Arbitrum: AccountChain = {
+  chainId: 42161,
+  name: "Arbitrum",
```
Comment (dcposch) [inline packages/daimo-common/src/chain.ts:60]: case > make consistent with `base`/`baseSepolia` above

File: packages/daimo-common/src/foreignToken.ts [hunk 4]
```diff
@@ -13,6 +35,15 @@ export type ForeignToken = {
   logoURI?: string;
 };
 
+export enum TokenLogo {
+  ETH = "https://assets.coingecko.com/coins/images/279/large/ethereum.png",
+  USDC = "https://assets.coingecko.com/coins/images/6319/large/usdc.png",
+  USDT = "https://assets.coingecko.com/coins/images/325/large/tether.png?1547034089",
+  DAI = "https://assets.coingecko.com/coins/images/9956/large/dai-multi-collateral-mcd.png?1574218774",
+}
```
Comment (dcposch) [inline packages/daimo-common/src/foreignToken.ts:43]: 👍 

doesn't have to be in this PR but in future we can inline these as small base64 images

**Review by dcposch — APPROVED** [review]
Nice cleanup!

File: packages/daimo-common/src/chain.ts (hunk 3 above)
Comment (kayleegeorge) [reply packages/daimo-common/src/chain.ts]: good catch

**Pull request merged**